web/src/components/toggle.js

```diff
@@ -82,15 +82,11 @@ const Toggle = ({
 }) => {
   const [tooltipVisible, setTooltipVisible] = useState(false);
 
-  const activeIndex = findIndex(options, { value });
```

Comment:
I guess `findIndex` is not used here anymore, so it doesn't have to be imported (which reminds me, we should probably add a linting rule for that :wink:)